railties/lib/rails/application/configuration.rb

```diff
@@ -365,15 +366,27 @@ def database_configuration
         raise e, "Cannot load database configuration:\n#{e.message}", e.backtrace
       end
 
+      # Returns whether or not to use ANSI color codes when logging
+      # information. Defaults to +true+.
       def colorize_logging
         ActiveSupport::LogSubscriber.colorize_logging
       end
 
+      # Sets whether or not to use ANSI color codes when logging
+      # information.
       def colorize_logging=(val)
         ActiveSupport::LogSubscriber.colorize_logging = val
         generators.colorize_logging = val
       end
 
+      # Specifies what class to use to store the session. Possible values are
+      # +:cookie_store+ which is the default, +:mem_cache_store+, and
+      # +:disabled+. The last one tells Rails not to deal with sessions.
+      # Defaults to a cookie store with application name as the session key.
+      # Custom session stores can also be specified:
+      #
+      #   config.session_store :my_custom_store
+      #
```

Comment:
I updated the source text for this in https://github.com/rails/rails/pull/44423/files#diff-210d0f245e618b7f8c6fcb53f518bd7fd343ce7e5d8c609ecfdb149418beca19R336-R349.  Not sure if you want to "rebase" it.

I would lean toward leaving out the default value, since this method doesn't really have knowledge or responsibility for it.  But this class is a sort of "meta" API, so I can see an argument for either way.